jira-widget/jira.lua

```diff
@@ -167,13 +167,13 @@ local function worker(user_args)
 
         local cur_status = ''
         for _, issue in ipairs(result.issues) do
-            local path_to_avatar = HOME_DIR ..'/.cache/awmw/jira-widget/avatars/' .. issue.fields.assignee.name
 
+            local path_to_avatar = HOME_DIR ..'/.cache/awmw/jira-widget/avatars/' .. issue.fields.assignee.displayName
             if not gfs.file_readable(path_to_avatar) then
                 spawn.easy_async(string.format(
                         DOWNLOAD_AVATAR_CMD,
                         HOME_DIR,
-                        issue.fields.assignee.name,
+                        issue.fields.assignee.displayName,
```

Comment:
To support old Jira version, I think it would be nice to do something like:

```lua
issue.fields.assignee.name or issue.fields.assignee.displayName,
```
so that it takes non-nil value.
Could you please change it?

Reply:
Sure, thanks

Writing lua... I have no idea what I'm doing. I had a problem because lua throws if I try to concat a nil value.  Is what I wrote on the new commit the best way of doing this in lua?